Demos/02-extract/README.md

```diff
@@ -111,6 +111,19 @@ The next step is to use the Azure Data Factory to create a pipeline to extract t
 
     ![Screenshot of the Azure Data Factory menu](./../../Images/adfv2-setup-03.png)
 
+1. [Optional] By default the Azure Data Factory will use an **Integration Runtime** that is auto-resolving the region. As the Microsoft Graph Data Connect requires that your source and destination, and integration runtime to exist in the same Office 365 region, it is recommended that you create a new Integration Runtime with a fixed region.
+    
+    1. Click on *Connections*, at the bottom of the screen, then choose *Integration Runtimes*
+    1. Click on *New* and then select the first option (*Perform data movement and dispatch activities to external computes*), followed by *Next*.
+    1. Choose *Azure* as the environment and click *Next*
+    1. In the final screen fill in as follows and then click *Finish*.
```

Comment:
Use the following details to complete the form on the final screen & then select **Finish**: